src/pages/home/ReportScreen.js

```diff
@@ -195,10 +195,12 @@ class ReportScreen extends React.Component {
         // We create policy rooms for all policies, however we don't show them unless
         // - It's a free plan workspace
         // - The report includes guides participants (@team.expensify.com) for 1:1 Assigned
+        // - It has removed all guide participants before archiving
```

Comment:
I think we don't need a comment for this. It's making it more confusing
```suggestion
```

Reply:
Thank you, it has been updated